docs/modules/chat/examples/memory.ipynb

```diff
@@ -0,0 +1,161 @@
+{
```

Comment:
I think it would be worth it to show an example of using memory with a plain old `LLMChain` -- you don't **need** to use a `ConversationChain` to use memory and chat with an AI. It just enforces the `memory` variable. This notebook can stay, but might be worth it to add an simple llmchain + memory example in the chat quickstart notebook